python/src/Analytical_doc.i.in

```diff
@@ -178,3 +183,27 @@ Notes
 Performs the research of the design point and creates a
 :class:`~openturns.AnalyticalResult`, the structure result which is
 accessible with the method :meth:`getAnalyticalResult`."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::Analytical::getLimitStateTolerance
```

Comment:
I think Julien's earlier remarks were not taken into account, despite appearing as resolved. Maybe the relevant commits were lost?

Reply:
For me, the remark is about adding a new constructor of the class depending on this parameter. I removed the ctor.

Reply:
I meant the remarks about stating the [full name of the ResourceMap key](https://github.com/openturns/openturns/pull/2561/files#r1513021079) and about [replacing "Returns" with "Parameters"](https://github.com/openturns/openturns/pull/2561/files#r1513021873) in the "set" accessor.